pages/common/traefik.md

```diff
@@ -0,0 +1,19 @@
+# traefik
+
+> Traefik HTTP reverse proxy and load balance.
+
+- Start server with default config:
+
+`traefik`
+
+- Start server with custom config file:
+
+`traefik --c {{config_file}}.toml`
+
+- Start server with a cluster mode enabled:
+
+`traefik --cluster`
+
+- Start server with web ui enabled:
```

Comment:
ui => UI.

Also, the words "Start server" are kinda getting repetitive. Maybe you can just say "Enable web UI".

Reply:
1. I think it's important to dictate that the flag I'm referring to, both starts the Traefik server and enables the UI. While it's true that all of my items start the server, it's worth ensuring there is no confusion of altering a configuration to enable the web UI. It's a pretty short list, but a couple flags do not actually start the server.
2. This mirrors the same language on the nginx page. Nginx is pretty similar in terms of having a small number of args that don't start the server.

Reply:
>  It's a pretty short list, but a couple flags do not actually start the server.

I see. Okay then.